ansible/tasks/security/main.yml

```diff
@@ -1,90 +1,125 @@
     content: |
       #Type Path              Mode UID  GID  Age Argument
-      d     /dev/inst         0000 root root -   -
       d     /tmp/inst         0000 root root -   -
       d     /var/tmp/inst     0000 root root -   -
       d     /run/lock/inst    0000 root root -   -
 
-- name: Setup default security limits
-  block:
-  - name: Setup security limits.conf
-    copy:
-      dest: /etc/security/limits.conf
-      content: |
-        1000:65535           soft    nproc           150
-        1000:65535           hard    nproc           200
-        1000:65535           soft    nofile          4096
-        1000:65535           hard    nofile          10240
-  - name:
-    copy:
-      dest: /etc/security/limits.sh
-      mode: "a+x"
-      content: |
-        #!/bin/sh -e
-
-        PAM_UID=$(getent passwd "${PAM_USER}" | cut -d: -f3)
-
-        if [ "${PAM_UID}" -ge 1000 ]; then
-            /bin/systemctl set-property "user-${PAM_UID}.slice" \
-                           CPUQuota=50% MemoryLimit=512M BlockIOWeight=10
-        fi
-  - name:
-    copy:
-      dest: /etc/security/namespace.conf
-      content: |
-        /tmp      /tmp/inst/           user      root
-        /var/tmp  /var/tmp/inst/       user      root
-        /run/lock /run/lock/inst/      user      root
-  - name:
-    copy:
-      dest: /etc/security/namespace.init
-      content: |
-        #!/bin/sh -e
-        # It receives polydir path as $1, the instance path as $2,
-        # a flag whether the instance dir was newly created (0 - no, 1 - yes) in $3,
-        # and user name in $4.
-
-        if [ "$1" = "/dev" ]; then
-        	# If we are creating /dev
-        	if [ "$3" = 1 ]; then
-        		# Major and minor number for devices come from
-        		# https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/tree/Documentation/devices.txt
-        		mknod -m 666 /dev/null      char  1   3
-        		mknod -m 666 /dev/zero      char  1   5
-        		mknod -m 666 /dev/full      char  1   7
-        		mknod -m 666 /dev/random    char  1   8
-        		mknod -m 666 /dev/urandom   char  1   9
-        		mknod -m 666 /dev/fuse      char 10 229
-        		mknod -m 666 /dev/tty       char  5   0
-        		chown root:tty /dev/tty
-
-        		# Create devpts mountpoint
-        		mkdir -m 755 /dev/pts
-
-        		# Create the shm directory
-        		mkdir -m 1777 /dev/shm
-
-        		# Mandatory symlinks
-        		ln -s /proc/self/fd  /dev/fd
-        		ln -s fd/0           /dev/stdin
-        		ln -s fd/1           /dev/stdout
-        		ln -s fd/2           /dev/stderr
-        		ln -s null           /dev/X0R
-
-        		# Recommended symlinks
-        		ln -s /run/systemd/journal/dev-log /dev/log
-        	fi
-
-        	mount -o gid=5,mode=0620,ptmxmode=0666 \
-        		  -t devpts devpts /dev/pts
-        	ln -sf pts/ptmx /dev/ptmx
-        fi
-
-        exit 0
+
+- name: "pam_namespace:  Configure polyinstanciated directories"
```

Comment:
instantiated*

Reply:
Fixing that in a sec.  FWIW, you can just push spelling fixes to the branch, I won't mind.